docs/reference/Image.rst

```diff
@@ -263,8 +261,6 @@ This flips the input image by using the :data:`FLIP_LEFT_RIGHT` method.
 
 .. automethod:: PIL.Image.Image.verify
 
-.. automethod:: PIL.Image.Image.fromstring
```

Comment:
`PIL.Image.Image.frombytes` is undocumented. Perhaps change this to `frombytes` instead of removing it.